tool/tsh/db.go

```diff
@@ -257,9 +282,39 @@ CA:        %v
 Cert:      %v
 Key:       %v
 `,
-		database.ServiceName, host, port, database.Username,
-		database.Database, profile.CACertPath(),
-		profile.DatabaseCertPath(database.ServiceName), profile.KeyPath())
+			database.ServiceName, host, port, database.Username,
+			database.Database, profile.CACertPath(),
+			profile.DatabaseCertPath(database.ServiceName), profile.KeyPath())
+	}
+	return nil
+}
+
+// onDatabaseConnect implements "tsh db connect" command.
+func onDatabaseConnect(cf *CLIConf) error {
+	tc, err := makeClient(cf, false)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	profile, err := client.StatusCurrent("", cf.Proxy)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	database, err := pickActiveDatabase(cf)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	cmd, err := getConnectCommand(cf, tc, profile, database)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	log.Debugf("Executing command: %v %v", cmd.Path, strings.Join(cmd.Args[1:], " "))
```

Comment:
nit: we can use  the `func (c Cmd) String()` method here:
```suggestion
	log.Debugf("Executing command: %s", cmd)
```